generator/lib/behavior/i18n/I18nBehaviorObjectBuilderModifier.php

```diff
@@ -183,7 +183,7 @@ protected function addTranslatedColumnSetter(Column $column)
             $objectBuilder->addMutatorOpenOpen($functionStatement, $column);
         }
         $comment = preg_replace('/^\t/m', '', $comment);
-        $comment = str_replace('@return     ' . $i18nTablePhpName, '@return     ' . $tablePhpName, $comment);
+        $comment = str_replace('@return ' . $i18nTablePhpName, '@return ' . $tablePhpName, $comment);
         $functionStatement = preg_replace('/^\t/m', '', $functionStatement);
```

Comment:
The str_replace() doesn't work because the needle doesn't match the comment it should be replacing.
This results in the wrong @return type (eg. ModelI18n instead of Model) which messes up the fluent API in supporting IDE's.